src/pages/Game.tsx

```diff
@@ -71,6 +73,10 @@ const GamePage = ({ ws, currentUser }: GameProps): JSX.Element => {
       const payload = params as Record<string, unknown>;
       ws?.sendMessage(OutgoingAction.PLAYED_TILE_INTERACTION, { gameId: game.gameId, ...payload });
     },
+    [OutgoingAction.SELF_PLAY_TILE]: (params: unknown) => {
+      const payload = params as Record<string, unknown>;
+      ws?.sendMessage(OutgoingAction.PLAYED_TILE_INTERACTION, { gameId: game.gameId, ...payload });
```

Comment:
should it be **OutgoingAction.SELF_PLAY_TILE**?
```ts
ws?.sendMessage(OutgoingAction.SELF_PLAY_TILE, { gameId: game.gameId, ...payload });
```

Reply:
Good catch. Error due to copy pasta